brut.apktool/apktool-lib/src/main/java/brut/androlib/Androlib.java

```diff
@@ -486,6 +488,25 @@ public boolean buildResourcesFull(File appDir, UsesFramework usesFramework)
                     }
                 }
 
+                if (buildOptions.netSecConf) {
+                    MetaInfo meta = readMetaFile(new ExtFile(appDir));
+                    if (meta.sdkInfo != null && meta.sdkInfo.get("targetSdkVersion") != null) {
+                        if (Integer.parseInt(meta.sdkInfo.get("targetSdkVersion")) < 24) {
+                            LOGGER.warning("Target SDK version is lower than 24! Network Security Configuration might be ignored!");
+                        }
+                    }
+                    File netSecConfOrig = new File(appDir, "res/xml/network_security_config.xml");
+                    if (netSecConfOrig.exists()) {
+                        netSecConfOrig.delete();
+                    }
```

Comment:
Should we expose a message to console that the existing config is being removed for an assumed less-restrictive one?

Just thinking this might be one of the larger features that Apktool would intentionally change/remove files.

Reply:
added that -- let me know if you have something more verbose in mind